### DISCUSSION TITLE
Not able to create Branch and PR for github graduation 2022

### DISCUSSION THREAD

**Discussion by ptoshniasu** [community/community#17715] [GitHub Education] [answered]
Getting this error - 
remote: Permission to education/GitHubGraduation-2022.git denied to ptoshniasu.
fatal: unable to access 'https://github.com/education/GitHubGraduation-2022.git/': The requested URL returned error: 403

**Comment by Steffo99** [comment, marked as answer]
You need to create your own fork first, and then use that as the remote.

**Comment by mlarasusan** [comment]
Hello [ptoshniasu](https://github.com/ptoshniasu), were you able to solve it? If not, let me know and I'll help you. Thanks :]

**Reply by ptoshniasu** [reply]
@mlarasusan I can definitely use some help here. Let me know if we can connect further here is my mail id: ptoshni2@asu.edu

**Reply by mlarasusan** [reply]
Can you share your PR here in the discussions? I'll take a look. Thanks :]

**Reply by ptoshniasu** [reply]
@mlarasusan Thank you for your quick reply - the issue is I have created my branch from main, added the required content but I am not able to push my branch to repo and that's the error I am seeing in my console - 

git push -u origin ptoshniasu
Username for 'https://github.com/education/GitHubGraduation-2022.git': ptoshniasu
Password for 'https://ptoshniasu@github.com/education/GitHubGraduation-2022.git':
remote: Permission to education/GitHubGraduation-2022.git denied to ptoshniasu.
fatal: unable to access 'https://github.com/education/GitHubGraduation-2022.git/': The requested URL returned error: 403

**Reply by mlarasusan** [reply]
I recommend you to try again. Make sure you're following the exact steps in the [repo description](https://github.com/education/GitHubGraduation-2022/).

**Reply by ptoshniasu** [reply]
@mlarasusan I tried n number of times, i don't know but I keep getting access right issue - 

git push -u origin ptoshniasu
ERROR: Permission to education/GitHubGraduation-2022.git denied to ptoshniasu.
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.

**Reply by ptoshniasu** [reply]
@mlarasusan No worries I am able to do it. I invested so much time to do it. The issue was I was creating branch in the Githubeducation repo and I was not forking it. Once I forked repo and created a branch it took me less than a minute. I hope it gets merged soon as I guess the 7500 count is already crossed there.
